Documentation/network/servicemesh/envoy-traffic-shifting.rst

```diff
@@ -0,0 +1,154 @@
+
+.. parsed-literal::
+
+    $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/servicemesh/envoy/helloworld-service-v1-v2.yaml
+
+
+Apply the ``envoy-helloworld-v1-90-v2-10.yaml`` file, which defines a ``CiliumEnvoyConfig``. 
+``CiliumEnvoyConfig`` can be used to load balance traffic destined to one Service to a
+group of backend Services. To load balance traffic to the helloworld Service, first create
+individual Services for each backend Deployment.
+
+.. parsed-literal::
+
+    $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/servicemesh/envoy/envoy-helloworld-v1-90-v2-10.yaml
+
+View information about these Pods and Service:
+
+.. code-block:: shell-session
+
+    $ kubectl get pods --show-labels -o wide
+    NAME                              READY   STATUS    RESTARTS   AGE   IP           NODE                       NOMINATED NODE   READINESS GATES   LABELS
+    client-64848f85dd-sjfmb          1/1     Running   0          2m23s   10.0.0.206   cilium-control-plane   <none>           <none>            kind=client,name=client,pod-template-hash=64848f85dd
+    helloworld-v1-5845f97d6b-gkdtk   1/1     Running   0          2m23s   10.0.0.241   cilium-control-plane   <none>           <none>            app=helloworld,pod-template-hash=5845f97d6b,version=v1
+    helloworld-v2-7d55d87964-ns9kh   1/1     Running   0          2m23s   10.0.0.251   cilium-control-plane   <none>           <none>            app=helloworld,pod-template-hash=7d55d87964,version=v2
+
+    $ kubectl get svc --show-labels
+    NAME            TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE   LABELS
+    helloworld      ClusterIP   10.96.194.77   <none>        5000/TCP   16m   app=helloworld,service=helloworld
+    helloworld-v1   ClusterIP   10.96.0.240    <none>        5000/TCP   4s    app=helloworld,service=helloworld,version=v1
+    helloworld-v2   ClusterIP   10.96.41.142   <none>        5000/TCP   4s    app=helloworld,service=helloworld,version=v2
+
+.. include:: warning.rst
+
+Try making several requests to the helloworld Service again.
+
+.. code-block:: shell-session
+
+    $ for i in {1..10}; do  kubectl exec -it $CLIENT -- curl  helloworld:5000/hello; done
+
+The test results are as follows::
+
+    Hello version: v1, instance: helloworld-v1-5845f97d6b-gkdtk
+    Hello version: v1, instance: helloworld-v1-5845f97d6b-gkdtk
+    Hello version: v1, instance: helloworld-v1-5845f97d6b-gkdtk
+    Hello version: v2, instance: helloworld-v2-7d55d87964-ns9kh
+    Hello version: v1, instance: helloworld-v1-5845f97d6b-gkdtk
+    Hello version: v1, instance: helloworld-v1-5845f97d6b-gkdtk
+    Hello version: v1, instance: helloworld-v1-5845f97d6b-gkdtk
+    Hello version: v1, instance: helloworld-v1-5845f97d6b-gkdtk
+    Hello version: v1, instance: helloworld-v1-5845f97d6b-gkdtk
+    Hello version: v1, instance: helloworld-v1-5845f97d6b-gkdtk
+
+The test results were as expected if you request the helloworld Service, 90% will send v1 and 10% will send v2.
```

Comment:
Can you change this to match the wording you used on lines 75-77?

Reply:
ok.done.